File: data-quality/trash/EmailAnalyserTest.scala
package it.gov.daf.dataquality.analysis

import org.scalatest.FunSuite

/**
  * @author Marco Bianchi (bianchi74@gmail.com)
  */
class EmailAnalyserTest extends FunSuite {

  val emailAnalyser = new EmailAnalyser

  //println(emailAnalyser.analyse("mbianchi@fub.it"))
//  println(emailAnalyser.analyse("mbianchi@fub.it"))
//  println(emailAnalyser.analyse("bianchi74@gmail.com"))

  //println(emailAnalyser.analyse("marco.bianchi@agid.gov.it"))
  //println(emailAnalyser.analyse("prot.gen.asl.vt.it@legalmail.it"))
  //println(emailAnalyser.analyse("aslumbria2@postacert.umbria.it"))
  println(emailAnalyser.analyse("affarigenerali.asprc@certificatamail.it"))

  //util.Map[String, String] analyse (emailAddress: String)  //test("Testing valid email addresses"){
    //assert(emailAnalyser.isValid("mbianchi@fub.it"))
    //assert(emailAnalyser.isValid("marco.bianchi@agid.gov.it"))


}

File: data-quality/src/test/java/it/gov/daf/dataquality/analysis/EmailAnalyserTest.java
package it.gov.daf.dataquality.analysis;

import junit.framework.TestCase;
import org.junit.Test;

/**
 * @author Marco Bianchi (bianchi74@gmail.com)
 */
public class EmailAnalyserTest extends TestCase {

    @Test
    public void testAnalyse() {
        EmailAnalyser emailAnalyser = new EmailAnalyser();

        //println(emailAnalyser.analyse("mbianchi@fub.it"))
        //println(emailAnalyser.analyse("mbianchi@fub.it"))
        //println(emailAnalyser.analyse("bianchi74@gmail.com"))

        //println(emailAnalyser.analyse("marco.bianchi@agid.gov.it"))
        //println(emailAnalyser.analyse("prot.gen.asl.vt.it@legalmail.it"))
        //println(emailAnalyser.analyse("aslumbria2@postacert.umbria.it"))
        System.out.println(emailAnalyser.analyse("affarigenerali.asprc@certificatamail.it"));

    }
}
